src/pybind/mgr/ssh/module.py

```diff
@@ -884,7 +884,7 @@ def _create_mon(self, host, network, name):
         # infer whether this is a CIDR network, addrvec, or plain IP
         if '/' in network:
             extra_args = ['--mon-network', network]
-        elif network.startswith('[v') and network.endswith(']'):
```

Comment:
We shouldn't allow '[' or ']' in the syntax as this will mess with shell escaping/regex.

Reply:
this is already the case with other ceph commands that take addrvecs.  The user just needs to use single quotes.

Reply:
this needs to be readded when https://github.com/ceph/ceph/pull/31672#issuecomment-554409119 is addressed.